src/Words.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class Words extends React.Component {
+    state = {
+        wordstyle: {
+           // textAlign:left ,
```

Comment:
its not good practice to leave commented unused code in your file when you commit